### PR TITLE
Fix #94: Move Elm code into src.

### LINKED ISSUE #94: Move Elm code into src
So that searching for Elm bits is easier.

### PR DESCRIPTION
**Pull request by n1k0**

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -3,16 +3,16 @@
   "version": "0.0.1",
   "description": "Firefox System Build info manager.",
   "scripts": {
-    "build": "elm-make Main.elm --output=build/app.js && npm run copy-assets",
-    "copy-assets": "copyfiles index.html style.css build/",
-    "debug": "elm-live Main.elm --output=app.js --debug",
+    "build": "npm run copy-assets && elm-make src/Main.elm --output=build/app.js",
+    "copy-assets": "rimraf build && mkdir build && cp -r public/* build",
```
Comment (magopian) [inline ui/package.json]: TIL about `rimraf`. Do you want to add it to the npm dependencies so we're sure it's available on all platforms? (Eg osx, windows...?)

File: ui/package.json [hunk 2]
```diff
@@ -3,16 +3,16 @@
...
+    "debug": "elm-live src/Main.elm --warn --dir=public/ --output=public/app.js --debug",
```
Comment (magopian) [inline ui/package.json]: :+1: for adding `--warn`

File: ui/package.json (hunk 1 above)
Comment (n1k0) [inline ui/package.json]: That's an excellent idea :)

File: ui/public/app.js [hunk 3]
```diff
@@ -0,0 +1,18420 @@
+
```
Comment (Natim) [inline ui/public/app.js]: I don't think we want to commit this file but rather add it in the .gitignore

Comment (n1k0) [inline ui/public/app.js]: Damn, you're right. Thanks for noticing!

**Pull request merged**